### ISSUE TITLE
Is it a good idea to add operator, operator:wikidata, operator:wikipedia?

### ISSUE THREAD

**Issue by matkoniecz** [osmlab/name-suggestion-index#2471]
I think that given that the tags are the same everywhere it may be better to put it directly into Wikidata, to make updates easier - single edit, rather than editing tens or hundreds of OSM objects.

@Adamant36

**Comment by Adamant36** [comment]
Huh? I'm not clear on the question. 

Personally, the main reason I add the operator and operator Wikipedia information is because it keeps people from accidentally putting it in as the brand:wikipedia information when they shouldn't. Its also helped me to find names in Taginfo that were wrong. I think if OSM uses it and the point the index is to help improving tagging then its worth having them, because it leaves one less value up to bad tagging in that case. Plus, it just keeps us from having to discuss repeatedly what goes in the brand/brand:wikipedia entries or not repeatedly. We don't have to make a possible sacrifice that way. I think mappers find the information interesting also. Although I could be wrong.

**Comment by bhousel** [comment]
> Personally, the main reason I add the operator and operator Wikipedia information is because it keeps people from accidentally putting it in as the brand:wikipedia information when they shouldn't. Its also helped me to find names in Taginfo that were wrong.

These are good reasons..  I somewhat think it is not super necessary to add them, but it doesn't hurt anything either, so if people want to look up and add the tags, I don't have a problem with it.  As long as the data is correct, that's what matters.

**Comment by 1ec5** [comment]
Note that we’d be able to predetermine the operator only in certain cases, where the chain is completely company owned-and-operated (e.g., Walmart), but not in the case of a franchise (e.g., Shell).

**Comment by Adamant36** [comment]
@1ec5, what is it in cases of franchises? Owner? That doesn't seem right because the operator is also the owner.

**Comment by 1ec5** [comment]
`operator` is the right tag in the case of a franchise, too, but name-suggestion-index doesn’t automatically know the franchisee’s name or QID just based on the franchiser.

**Comment by Adamant36** [comment]
I meant what's the proper to tag whoever the franchising company is. Instead of the franchiser. Which would be the operator. For instance with Popeye's Wikipedia says Restaurant Brands International is the parent company. They franchise out Popeye's. They wouldn't be owner, that would be the person who owns the franchise. There's no parent=* tag in OSM though. So I'm not sure what an alternative to the operator or owner tags would be.

**Comment by bhousel** [comment]
OpenStreetMap doesn't really have a tag for "parent company" (franchisor).   That information can come from Wikidata anyway, if anyone wants it.

The `operator` tag is used for whatever entity actually operates the business (franchisee or parent company, depending on the situation).  I think it's kind of silly to record this information, but some people do it.

**Comment by 1ec5** [comment]
Yeah, I was conflating franchisor with brand for the purpose of this discussion. Wikidata is a more appropriate place to detail organization structures, which can get quite complex.

It can be useful for mappers to tag the franchisee, because in some industries, a franchisee may switch affiliations en masse, use a variant of the usual logo, have distinct menu choices, etc. (For example, UDF used to operate Mobil stations until they switched to their own brand a decade or so ago: #2234.) In the foodservice and fuel distribution industries, a franchisee can usually be verified from a publicly posted plaque, or at least a receipt. But that would be up to the mapper on a case-by-case basis.

**Comment by Adamant36** [comment]
"OpenStreetMap doesn't really have a tag for "parent company" (franchisor)"

I didn't think they did. 

"That information can come from Wikidata anyway, if anyone wants it." 

Having it on Wikidata doesn't keep people from repeatedly entering the parent companies Wikipedia information into the brand:wikipedia key though. Which is really the only reason I add it. 

It would probably be helpful if you mention the difference between a brand and operator in the FAQ when it make it so that wont happen as much. Maybe you can put in a side note that people can put it in Wikidata if they want, by using the "subsidiary" or whatever it is label.
